conditions/README.md

```diff
@@ -0,0 +1,341 @@
+            }
+          }
+        ]
+      }
+    ],
+    "identifier": [
+      {
+        "system": "https://PSSAdaptor/B83002",
+        "value": "BF627285-8E57-46C7-BBAF-27AFBC7C23B8"
+      }
+    ],
+    "clinicalStatus": "active",
+    "category": [
+      {
+        "coding": [
+          {
+            "system": "https://fhir.hl7.org.uk/STU3/CodeSystem/CareConnect-ConditionCategory-1",
+            "code": "problem-list-item",
+            "display": "Problem List Item"
+          }
+        ]
+      }
+    ],
+    "code": {
+      "coding": [
+        {
+          "extension": [
+            {
+              "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-coding-sctdescid",
+              "extension": [
+                {
+                  "url": "descriptionId",
+                  "valueId": "1488801013"
+                },
+                {
+                  "url": "descriptionDisplay",
+                  "valueString": "H/O: aspirin allergy"
+                }
+              ]
+            }
+          ],
+          "system": "http://snomed.info/sct",
+          "code": "395102008",
+          "display": "H/O: aspirin allergy"
+        }
+      ],
+      "text": "H/O: aspirin allergy"
+    },
+    "subject": {
+      "reference": "Patient/dc9cc923-60ed-40df-8a3e-64dc6bc89bf2"
+    },
+    "onsetDateTime": "2010-01-13",
+    "assertedDate": "2010-01-13T11:41:26+00:00",
+    "asserter": {
+      "reference": "Practitioner/1E473786-E7FA-785E-C911-A8D38FB56F20"
+    },
+    "note": [
+      {
+        "text": "Unspecified Significance: Defaulted to Minor"
+      },
+      {
+        "text": "Drug Allergy - Apsrin"
+      },
+      {
+        "text": "Active Problem, Not Significant (Minor)"
+      }
+    ]
+  }
+}
+```
+</details>
+
+1. If `LinkSet / conditionNamed / namedStatementRef` is present. Could reference one of the resource types stated [here](https://developer.nhs.uk/apis/gpconnect-1-6-0/accessrecord_structured_problems.html#extensionactualproblem)
+2. If `LinkSet / component [index] / statementRef / id [@root]` is present. Could reference another Condition or one of the resource types stated [here](https://developer.nhs.uk/apis/gpconnect-1-6-0/accessrecord_structured_problems.html#extensionrelatedclinicalcontent)
+3. If `LinkSet / component / statementRef / id [@root]` references a Link Set or another Link Set references the mapped Link set in the same manner.  
+
+
+### notes
+
+The following
+
+- fixed value = `"Defaulted status to active : Unknown status at source"` if `linkSet / code [@code]` does not equal `394774009` (active) or `394775005` (inactive)
+
+## Unmapped fields
+
+- implicitRules
+- language
+- text
+- contained
+- extension
+- modifierExtension
+- identifier.use
+- identifier.type
+- identifier.type.coding
+- active
+- name
+- gender
+- birthDate
+- deceasedDateTime
+- address
+- maritalStatus
+- multipleBirthBoolean
+- generalPractitioner
+- link
```

Comment:
What source did you use for this list of fields? I'm struggling to find some of these.

Reply:
I don't know where these have come from either. I'll remove the unmapped fields section as all the elements sated in https://developer.nhs.uk/apis/gpconnect-1-6-0/accessrecord_structured_problems.html are used.